tests/zfs-tests/tests/functional/mmp/mmp.cfg

```diff
@@ -38,3 +38,9 @@ export MMP_HISTORY_OFF=0
 export MMP_INTERVAL_HOUR=$((60*60*1000))
 export MMP_INTERVAL_DEFAULT=1000
 export MMP_INTERVAL_MIN=100
+
+export MMP_IMPORT_INTERVALS=10
+export MMP_FAIL_INTERVALS_DEFAULT=5
+export MMP_FAIL_INTERVALS_MIN=2
```

Comment:
These values should be updated to reflect the new larger defaults when you rebase this PR on master.  Which you should do when refreshing this.

Reply:
Fixed